blog/pymc3_vs_pymc_v4.ipynb

```diff
@@ -6,44 +6,47 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    data = pd.read_csv("/Users/twiecki/personal/homepage/WhileMyMCMCGentlySamples/content/radon.csv")
</pre><p> </p><p>Change this to be a load from github, as thisll break anywhere thats not your computer, radon csv is readily available in our repo</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc.io/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>